ui/lib/core/addon/templates/components/replication-dashboard.hbs

```diff
@@ -1,23 +1,46 @@
 <div class="replication-dashboard box is-sideless is-fullwidth is-marginless">
-  <h2 class="has-text-weight-semibold is-size-4 has-bottom-margin-xs">Primary Cluster</h2>
-  <p class="has-text-grey is-size-8">If you set a primary_cluster_addr when enabling replication, it will appear here.</p>
+  {{#if isSecondary}}
```

Comment:
something to revisit in another PR, but I kind of remember there being a spacing issue here when this conditional was not true, e.g. !isSecondary.